### ISSUE TITLE
Compendium Fantasy League - no points/level after Play-offs

### ISSUE THREAD

**Issue by katwin** [ValveSoftware/Dota2-Gameplay#13062]
### Description

I didn’t received any level for Play-offs points for Fantasy League - now it’s October,25th. Part of my steam friends had received it a few days ago. When can I get Compendium level and finalised Play-offs Fantasy results for my prediction?

### Screenshots

![EF145B72-9A1D-40D4-8FA6-055C306F3A46](https://github.com/ValveSoftware/Dota2-Gameplay/assets/26935760/1adfb46e-7773-4a8a-9113-fe7c4771b51d)

**Comment by ligaapl** [comment]
Yes, me too :(

**Comment by aggoo** [comment]
I have the same problem, is it weird that I also have team liquid in my fantasy?
![3](https://github.com/ValveSoftware/Dota2-Gameplay/assets/38524176/d3ff786a-93aa-4208-95ba-a46fab3fab24)

**Comment by SerCoco** [comment]
Same problem here
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/149002258/dac8226d-2d3a-4420-997d-fceebd6be791)

**Comment by HEGATNB** [comment]
same
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/149002661/d540f30b-b93d-4a0d-8f17-329610c60cd1)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/149002661/98c9d4d5-e3f2-4063-b69b-d19ab4c93cf6)

**Comment by whysoshorthuh** [comment]
yeah, Valve, pls fix
1040031623

**Comment by Lfey-gay** [comment]
такая же проблема, ID в доте:  845455429
![HVqRV1Tp1-k](https://github.com/ValveSoftware/Dota2-Gameplay/assets/149004608/f623238b-f6e8-41ee-a622-4b3ab7f24a2b)

**Comment by EmreBaltacii** [comment]
do they fix when you send your ID?

**Comment by goges9** [comment]
the third stage will start in 20 hours and i still don't get my points, volvo wtf

**Comment by uknwnth** [comment]
same here. my id: 27542470

**Comment by whysoshorthuh** [comment]
> do they fix when you send your ID?

Idk, I hope so

**Comment by goges9** [comment]
![20231025234844_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/36965883/10be49d2-ca58-4501-b4dc-c54418304ac2)
ok % appeared, but there is 0% and points still not show up

**Comment by erxor-github** [comment]
<img width="1920" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/107615194/a11db584-8156-412f-9dc4-a6924529e3d9">
no more TBD after coordinator update. now says 0.00% but still no rewards. 1 day left to gain the extra spins for next fantasy

**Comment by EmreBaltacii** [comment]
yes, FRIENDS list is updated and Percentile appears now (%0 is a bug obv) but still no points received either...

**Comment by jolyanjanpol** [comment]
Hi same here. My friends already have it but still no claim reward for me. 1254902124

**Comment by gruianovac** [comment]
why havent fantasy rewards been given? the roster will be locked soon ?!
those levels and rerolls are very important to get immediately, and it is not immediate, it has been a WQEEK

40345246

**Comment by EmreBaltacii** [comment]
Its given to me right now. Check again guys
